src/tools/stories.ts

```diff
@@ -322,6 +322,13 @@ Iteration: ${story.iteration_id ? `${story.iteration_id}` : "[None]"}
 Description:
 ${story.description}
 
+External links:
+${
+	story.external_links && story.external_links.length > 0
+		? story.external_links.join("\n")
```

Comment:
Could we make this an unordered list to match other fields?

```suggestion
		? story.external_links.map(link => `- ${link}`).join("\n")
```